ports/cxxgraph/portfile.cmake

```
@@ -0,0 +1,13 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO ZigRazor/CXXGraph
+    REF v2.0.0

```

```suggestion
    REF "v{VERSION}"
```
Please use the new function.